src/index.js

```diff
@@ -229,11 +235,19 @@ if ( !! window.wcCalypsoBridge.isEcommercePlan ) {
 	}
 }
 
+// Remove plugins that had already been added.
+const taskPlugins = getPlugins( 'woocommerce-tasks' );
+taskPlugins.forEach( ( plugin ) => {
+	if ( pluginsToRemove.includes( plugin.name ) ) {
+		unregisterPlugin( plugin.name );
+	}
+} );
```

Comment:
This is done on every page load? Is that required?

Reply:
Yes it's done on every page load, AFAIK plugin in JS needs to be loaded in every page load.